clients/admin-ui/src/features/privacy-experience/privacy-experience.slice.ts

```diff
@@ -0,0 +1,109 @@
+import { createSelector, createSlice } from "@reduxjs/toolkit";
+
+import type { RootState } from "~/app/store";
+import { baseApi } from "~/features/common/api.slice";
+import {
+  Page_PrivacyExperienceResponse_,
+  PrivacyExperience,
+  PrivacyExperienceResponse,
+  PrivacyNoticeRegion,
+} from "~/types/api";
+
+export interface State {
+  page?: number;
+  pageSize?: number;
+}
+
+const initialState: State = {
+  page: 1,
+  pageSize: 10,
```

Comment:
Should the pagesize be a small number like 10 when there is no way to switch between pages?

Reply:
good idea, I'll increase to 50 😎